src/pages/Manager.vue

```diff
@@ -128,6 +128,7 @@
         <DownloadProgressModal />
         <DownloadModVersionSelectModal />
         <UpdateAllInstalledModsModal />
+        <LaunchTypeModal/>
```

Comment:
Would it be more hassle than it's worth to use `v-if` and render the component only on target OS?